templates/zerver/app/navbar_alerts.html

```diff
@@ -47,9 +47,17 @@
     </div>
     <div data-process="bankruptcy" class="alert alert-info brankruptcy">
         <div data-step="1">
-            {% trans count=page_params.unread_msgs.count %}
-            Welcome back!  You have <span class="bankruptcy_unread_count">{{ count }}</span> unread messages.  Do you want to mark them all as read?
-            {% endtrans %}
+            {% with unread_initialize= page_params.unread_msgs.old_unreads_missing, count=page_params.unread_msgs.count %}
+                {% if unread_initialize %}
```

Comment:
This variable name is confusing; I think we want `old_unreads_missing` as the name here.

Reply:
Sure.